utils.js

```diff
@@ -8465,6 +8465,20 @@ export const getNugetMetadata = async function (
             p.license = findLicenseId(body.catalogEntry.licenseExpression);
           } else if (body.catalogEntry.licenseUrl) {
             p.license = findLicenseId(body.catalogEntry.licenseUrl);
+            // as the old .NET framework is not open source and therefore the license not in the SPDX index,
+            // the license info is provided with name ".NET Library License"
+            // resolved only here as this license should only be related to Nuget projects
+            if (
+              p.license &&
+              typeof p.license === "string" &&
+              (p.license.endsWith("?LinkId=329770") ||
+                p.license.endsWith("dotnet_library_license.htm"))
+            ) {
+              p.license = {
```

Comment:
Can we move this logic to getLicenses to make the tests pass? https://github.com/CycloneDX/cdxgen/blob/master/utils.js#L202